src/annotator/util/shadow-root.js

```diff
@@ -2,9 +2,13 @@
  * Load stylesheets for annotator UI components into the shadow DOM root.
  */
 function loadStyles(shadowRoot) {
-  const adderStyles = Array.from(document.styleSheets)
-    .map(sheet => sheet.href)
-    .filter(url => (url || '').match(/annotator\.css/));
+  const url = /** @type {HTMLLinkElement|undefined} */ (document.querySelector(
+    'link[rel="stylesheet"][href*="/build/styles/annotator.css"]'
```

Comment:
This URL is currently relative to the root of client's npm package (see https://unpkg.com/browse/hypothesis@1.605.0/). Currently that will work with all of the ways that we package the client (development server, https://cdn.hypothes.is, browser extension) but we may want to change that in future.

Reply:
👍🏼